Library/common.props

```diff
@@ -31,8 +33,24 @@
   <!-- Constants -->
   <PropertyGroup>
     <DefineConstants Condition="'$(TargetFramework)' == 'netstandard1.5'">$(DefineConstants);NETCORE</DefineConstants>
+    <DefineConstants Condition="'$(TargetFramework)' == 'netstandard2.0'">$(DefineConstants);NETCORE</DefineConstants>
     <DefineConstants Condition="'$(TargetFramework)' == 'net20'">$(DefineConstants);NET20</DefineConstants>
     <DefineConstants Condition="'$(TargetFramework)' == 'net40'">$(DefineConstants);NET40</DefineConstants>
     <DefineConstants Condition="'$(TargetFramework)' == 'net45'">$(DefineConstants);NET45</DefineConstants>
   </PropertyGroup>
+
+  <!-- .NET dependencies -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net20' ">
```

Comment:
All the .NET dependencies are now in common.props.

Reply:
Ah.. but that adds `System.Xml` to all projects. Looking through it, a number of projects don't reference `System.Xml` now, such as `DiscUtils.Vhdx`, `DiscUtils.Vmdk`, ..

Reply:
.. common.props doesn't have System.Xml .. It wasn't needed?

Reply:
Nope, there's a bunch of references which are included by default - see [Microsoft.NET.Sdk.BeforeCommon.targets](https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets). So all these references can be removed; I've updated the PR accordingly.